persist-cli/src/main/java/io/ballerina/persist/cmd/Generate.java

```diff
@@ -139,6 +137,13 @@ public void execute() {
                     generatedSourceDirPath = Paths.get(this.sourcePath, BalSyntaxConstants.GENERATED_SOURCE_DIRECTORY,
                             entityModule.getModuleName());
                 }
+
+                if (!Files.exists(generatedSourceDirPath)) {
+                    errStream.printf("The generated source directory: %s doesn't exist. " +
+                                    "Please run `bal persist init` to initiate the project before generation.%n",
```

Comment:
```suggestion
                    errStream.printf("The generated source directory: %s doesn't exist. " +
                                    "Run `bal persist init` to initiate the project before generation.%n",
```

Reply:
Shall we use lower case for the error messages? Seems thats the convention we follow for all ballerina error messages.